pkg/kubectl/cmd/annotate.go

```diff
@@ -41,39 +41,6 @@ import (
 	"k8s.io/kubernetes/pkg/kubectl/util/i18n"
 )
 
-// AnnotateOptions have the data required to perform the annotate operation
-type AnnotateOptions struct {
```

Comment:
i see several definition moved like this. is it necessary?

Reply:
Hi Kim
It was not necessary but while editing, I observed the pattern of defining vars first, then structs. So I moved them to follow the pattern. Shall I revert them?

Reply:
> Shall I revert them?

yeah. especially for such "wide" pulls. not only easier for review, but also helpful for sorting blame msg. plz keep a minimal change.

Reply:
Done.
Please review and let me know if there's something else I should keep in mind.